src/it/scala/io/scalac/tezos/translator/EmailSenderSpec.scala

```diff
@@ -0,0 +1,105 @@
+package io.scalac.tezos.translator
+
+import akka.actor.ActorSystem
+import akka.event.LoggingAdapter
+import akka.testkit.TestKit
+import com.dimafeng.testcontainers.{ForEachTestContainer, MySQLContainer}
+import com.icegreen.greenmail.util.{GreenMail, GreenMailUtil, ServerSetupTest}
+import io.scalac.tezos.translator.actor.EmailSender
+import io.scalac.tezos.translator.config.{Configuration, CronConfiguration, EmailConfiguration}
+import io.scalac.tezos.translator.model.{SendEmailDTO, SendEmailModel}
+import io.scalac.tezos.translator.repository.Emails2SendRepository
+import io.scalac.tezos.translator.routes.JsonHelper
+import io.scalac.tezos.translator.service.Emails2SendService
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{Matchers, WordSpecLike}
+import slick.jdbc.MySQLProfile
+
+import scala.concurrent.duration._
+import scala.concurrent.{ExecutionContextExecutor, Future}
+import scala.language.postfixOps
+
+class EmailSenderSpec
+  extends TestKit(ActorSystem("MySpec"))
+  with WordSpecLike
+  with ScalaFutures
+  with Matchers
+  with JsonHelper
+  with ForEachTestContainer {
+
+    implicit val ec: ExecutionContextExecutor = system.dispatcher
+    override lazy val container = MySQLContainer()
+    override implicit val patienceConfig: PatienceConfig = PatienceConfig(10 seconds)
+
+    private trait DatabaseFixture extends DbTestBase {
+      val testDb: MySQLProfile.backend.Database = DbTestBase.dbFromContainer(container)
+
+      val email2SendService = new Emails2SendService(emails2SendRepo, testDb)
+
+      recreateTables()
+      greenMail.setUser(testMailUser, testMailPass)
+      greenMail.start()
+    }
+
+    val log: LoggingAdapter = system.log
+    val emails2SendRepo = new Emails2SendRepository
+    val greenMail = new GreenMail(ServerSetupTest.SMTP)
+    val testMailUser = "sender@scalac.io"
```

Comment:
usually I suffix mail account with more ids, so it can be easier to filter it if by some mistake real mailer will be used e.g.
`sender@scalac.io` to `sender-tezostests@scalac.io`